src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/response/ScheduleItem.kt

```diff
@@ -0,0 +1,100 @@
+data class ScheduleItem(
+  @Schema(description = "The internal ID for the video booking", example = "123")
+  val videoBookingId: Long,
+
+  @Schema(description = "The internal ID for an appointment related to a booking. It is unique in this list", example = "123")
+  val prisonAppointmentId: Long,
+
+  @Schema(description = "The booking type", allowableValues = ["COURT", "PROBATION"], example = "COURT")
+  val bookingType: BookingType,
+
+  @Schema(description = "The booking status", allowableValues = ["ACTIVE", "CANCELLED"], example = "ACTIVE")
+  val statusCode: BookingStatus,
+
+  @Schema(description = "The video link URL to attend this event", example = "https://video.link.url")
+  val videoUrl: String?,
+
+  @Schema(description = "The comments provided for the booking", example = "Free text comment")
+  val bookingComments: String?,
+
+  @Schema(description = "True if the booking was made by a prison user", allowableValues = ["true", "false"], example = "false")
+  val createdByPrison: Boolean,
+
+  @Schema(description = "The internal court ID, if this is a court booking", example = "1234")
+  val courtId: Long?,
+
+  @Schema(description = "The court code, if this is a court booking", example = "DRBYMC")
+  val courtCode: String?,
+
+  @Schema(description = "The court description, if this is a court booking", example = "Derby Magistrates")
+  val courtDescription: String?,
+
+  @Schema(description = "The court hearing type code, if this is a court booking", example = "APPEAL")
+  val hearingType: CourtHearingType?,
+
+  @Schema(description = "The court hearing type description, if this is a court booking", example = "Appeal hearing")
+  val hearingTypeDescription: String?,
+
+  @Schema(description = "The internal probation team ID, if this is a probation booking", example = "1234")
+  val probationTeamId: Long?,
+
+  @Schema(description = "The internal probation team code, if this is a probation booking", example = "BLCKPPP")
+  val probationTeamCode: String?,
+
+  @Schema(description = "The probation team description, if this is a probation booking", example = "Blackpool PP")
+  val probationTeamDescription: String?,
+
+  @Schema(description = "The probation meeting type code, if this is a probation booking", example = "PSR")
+  val probationMeetingType: ProbationMeetingType?,
+
+  @Schema(description = "The probation meeting type description, if this is a probation booking", example = "Pre-sentence report")
+  val probationMeetingTypeDescription: String?,
+
+  @Schema(description = "The prison code", example = "MDI")
+  val prisonCode: String,
+
+  @Schema(description = "The prison name", example = "HMP Moorland")
+  val prisonName: String,
+
+  @Schema(description = "The prisoner number (NOMS ID)", example = "A1234AA")
+  val prisonerNumber: String,
+
+  @Schema(
+    description = "The appointment type",
+    allowableValues = ["VLB_COURT_PRE", "VLB_COURT_MAIN", "VLB_COURT_POST", "VLB_PROBATION"],
```

Comment:
Same here.  Enum will take of allowable values.

Reply:
Removed the allowableValues.